.travis.yml

```diff
@@ -10,6 +10,7 @@ matrix:
   fast_finish: true
   include:
     - php: 5.3
+      dist: precise
```

Comment:
because https://docs.travis-ci.com/user/reference/trusty#PHP-images

Reply:
This is already fixed in 6.0.0-dev.